byoqm/source_repository/query_translations.py

```diff
@@ -5,6 +5,10 @@
         "bool_operator_child": "boolean_operator",
         "comment": "(comment)",
         "function": "(function_definition)",
+        "if_statement": "(if_statement)",
+        "for_statement": "(for_statement)",
+        "while_statement": "(while_statement)",
+        "catch_statement": "(except_clause)",
```

Comment:
Just for my own understanding - What about the try part of this statement? Does that not have its own unique keyword?

Reply:
The paper for cognitive complexity only says to increment whenever there is a catch clause (or in python world, except). So I saw no reason to look for try.